src/Dropdown.php

```diff
@@ -3183,7 +3183,7 @@ public static function getDropdownValue($post, $json = true)
                 ];
 
                 if (
-                    $_SESSION['glpiis_ids_visible'] && (int) $post['searchText'] === $post['searchText']
+                    $_SESSION['glpiis_ids_visible'] && (int) $post['searchText'] == $post['searchText']
```

Comment:
If we want to check whether the searched text is only containing numbers, we could do this check :
```suggestion
                    $_SESSION['glpiis_ids_visible'] && preg_match('/^\d+$/', $post['searchText']) === 1
```